controllers/index.js

```diff
@@ -1,9 +1,21 @@
 var getWeather = require('../services/getWeather');
 var Controller = {};
 var Async = require('async');
+var geoip = require('geoip-lite');
 
 Controller.index = function(req, response) {
-  var params = req.query;
+  var params = getParams();
+
+  function getParams() {
+    if(req.query.lat !== undefined && req.query.lon !== undefined) {
```

Comment:
I would pull this `if` statement block into a variable. Something like:
```
var ifLatAndLong = req.query.lat !== undefined && req.query.lon !== undefined;
```
It might feel like a bit overkill, but I find it really helpful when I'm jumping into code or I have a lot of logic going on.